.babelrc

```diff
@@ -1,3 +1,3 @@
 {
-  "presets": ["react", "es2015", "stage-0"]
+  "presets": ["es2015", "react", "flow", "stage-0"]
```

Comment:
The 'flow' preset is required now because it was removed from the 'react' preset.